renderapi/transform/leaf/thin_plate_spline.py

```diff
@@ -411,3 +409,85 @@ def adaptive_mesh_estimate(
                 max_iter=max_iter,
                 nworst=nworst,
                 niter=0)
+
+    @staticmethod
+    def src_array(xmin, ymin, xmax, ymax, nx, ny):
+        """create N x 2 array of regularly spaced points
+
+        Parameters
+        ----------
+        xmin : float
+            minimum of x grid
+        ymin : float
+            minimum of y grid
+        xmax : float
+            maximum of x grid
+        ymax : float
+            maximum of y grid
+        nx : int
+            number of points in x axis
+        ny : int
+            number of points in y axis
+
+        Returns
+        -------
+        src : :class:`numpy.ndarray`
+            (nx * ny) x 2 array of coordinated.
+
+        """
+        x = np.linspace(xmin, xmax, nx)
+        y = np.linspace(ymin, ymax, ny)
+        xt, yt = np.meshgrid(x, y)
+        src = np.vstack((xt.flatten(), yt.flatten())).transpose()
+        return src
```

Comment:
Not really a suggestion, but there is a one-liner to do this (with a different ordering) using mgrid's complex step length:
`numpy.mgrid[ymin:ymax:ny*1j, xmin:xmax:nx*1j].reshape(2, -1).T`